.github/workflows/macos-build-and-deploy.yml

```diff
@@ -6,6 +6,12 @@ on:
       - master
     tags:
       - 'v[0-9]+.[0-9]+.[0-9]+'
+  workflow_dispatch: # Allows you to run this workflow manually from the Actions tab
+    inputs:
+      redis-ref:
```

Comment:
Where do you use this input?

Reply:
copy pasted from another flow. it is not used in this flow